frontend/src/app/shared/community-agreement/community-agreement.widget.html

```diff
@@ -0,0 +1,209 @@
+<mat-card-content class="content-container">
+  <p *ngIf="!has_user_agreed" style="margin-left: 15px; margin-top: 10px">
+    ***Please read and accept the community agreement below to continue to CSXL
+    web page.***
```

Comment:
This currently looks a little half-baked with the asterisks in paragraph tags. Can we elevate this to be the header of the dialog using the `mat-dialog-header`: https://material.angular.io/components/dialog/examples#dialog-content